frontend/src/scenes/apps/constants.tsx

```diff
@@ -1,30 +1,88 @@
 import { AppMetricsTab } from './appMetricsSceneLogic'
 
-export const DescriptionColumns: Record<
-    AppMetricsTab,
-    { successes: string; successes_on_retry?: string; failures: string }
-> = {
+interface Description {
+    successes: string
+    successes_tooltip?: React.ReactNode | string
+
+    successes_on_retry?: string
+    successes_on_retry_tooltip?: React.ReactNode | string
+
+    failures: string
+    failures_tooltip?: React.ReactNode | string
+}
+
+export const DescriptionColumns: Record<AppMetricsTab, Description> = {
     [AppMetricsTab.ProcessEvent]: {
         successes: 'Events processed',
+        successes_tooltip: (
+            <>
+                These events were successfully processed and transformed by the <i>processEvent</i> app method.
+            </>
+        ),
         failures: 'Failed events',
+        failures_tooltip: (
+            <>
+                These events had errors when being processed by the <i>processEvent</i> app method, but were still
```

Comment:
is that always true? For example with the down sampler do we return success?

Reply:
Yes this is true.